task11/Shipper.ts

```diff
@@ -0,0 +1,63 @@
+import { ShipmentType, STANDARD_PACKAGE_MAX_WEIGHT } from "./Shipment";
+
+export abstract class Shipper {
```

Comment:
I would suggest splitting this file to fore separate:

Shipper - our base abstract Class
AirEeastShipper
ChicagoSprintShipper
PacificParcelShipper

Obviously, we don't have a lot of code here, but the situation is always changeable and if we needed to expand all these classes, we would create a really huge file as a result 😄